config/tileset/elevation.json

```diff
@@ -451,6 +451,13 @@
       "minZoom": 9,
       "title": "Canterbury LiDAR 1m DEM (2020-2023)",
       "name": "canterbury_2020-2023_dem_1m"
+    },
+    {
+      "2193": "s3://nz-elevation/southland/southland_2020-2023/dem_1m/2193/",
+      "3857": "s3://linz-basemaps/elevation/3857/southland_2020-2023_dem_1m/01J9Q42DGRE9P73Y8ZRVJX2J4J/",
```

Comment:
This dataset already exists in the elevation config, need to update existing line rather than adding it a second time.

Reply:
Ah, I see, will do that now.

Reply:
Updated. Thanks.